src/data/companyData.ts

```diff
@@ -176,6 +176,28 @@ export function getLatestReportingPeriodWithEconomy(company: CompanyData) {
   return company.reportingPeriods.find(({ economy }) => Boolean(economy))
 }
 
+export function hasVerifiedScope3(emissions: Emissions | null | undefined): boolean {
+  if (
+    !emissions ||
+    !emissions.scope3 ||
+    !emissions.scope3.categories ||
+    // If all categories are null, then it's marked as not verified
+    !emissions.scope3.categories.some(
+      (category) => category.total !== null && category.total !== undefined
+    )
```

Comment:
We need to account for `scope3.statedTotalEmissions` here too.

If all scope3 categories are verified => total should be verified
If none scope 3 categories are verified => total should only be verified if statedTotalEmissions is verified